playwright/test/work.test.ts

```diff
@@ -21,7 +25,9 @@ const getWhereToFindItAndEncoreLink = async (page: Page) => {
 };
 
 const getAvailableOnline = async (page: Page) => {
```

Comment:
np; kind of wondering how useful this is? very np.

Reply:
As opposed to inlining the `getByRole` where it's required? I think it's about the same number of lines of code either way, and IMO it helps with readability a bit.

Reply:
Yeah I just wondered cause it awaited an await! Works for me either way.

Reply:
Those double awaits _are_ unnecessary, will remove.